components/ghost/ghost.app.js

```diff
@@ -0,0 +1,14 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "ghost_org_content_api",
+  methods: {
+    async getAuthors(page, latestId) {
+      const limit = 15;
+      return (results = await axios.get(
```

Comment:
Can you use axios request config instead of the unsafe url building?

https://github.com/axios/axios#request-config